packages/loot-core/src/shared/errors.ts

```
@@ -47,16 +61,16 @@ export function getDownloadError({ reason, meta, fileName }) {
 
     case 'out-of-sync-migrations':
       return (
-        'This budget cannot be loaded with this version of the app. ' +
-        'Make sure the app is up-to-date.'
+        t('This budget cannot be loaded with this version of the app. ') +
+        t('Make sure the app is up-to-date.')
       );
 
     default:
       const info = meta && meta.fileId ? `, fileId: ${meta.fileId}` : '';
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)